scaffolding-node/lib/scaffolding.sh

```diff
@@ -17,7 +17,11 @@ scaffolding_load() {
 }
 
 do_default_prepare() {
-  export NODE_ENV=production
+  # The NODE_ENV must be development in order to
+  # install the dev dependencies that are
+  # required to run build scripts
+  export NODE_ENV=development
```

Comment:
I think the default action is to install development dependencies so if you want to omit this entirely it would maintain the same behavior (I think)